src/packagedcode/pypi.py

```diff
@@ -517,6 +517,10 @@ def parse(cls, location, package_only=False):
         if license_file:
             extra_data['license_file'] = license_file
 
+        classifiers = project_data.get('classifiers', []) 
+        is_private = any('Private ::' in classifier for classifier in classifiers) 
+        extra_data['is_private'] = is_private
```

Comment:
We don't need to store these in `extra_data`, as we have a proper field for this at https://github.com/aboutcode-org/scancode-toolkit/blob/develop/src/packagedcode/models.py#L696 which I added sometime back here: https://github.com/aboutcode-org/scancode-toolkit/commit/f848741989d0d40309fc1220c58b5a9a145c897e See related issues and files for more examples on how it was added.